fuzzers/030-iob/generate.tcl

```diff
@@ -53,20 +53,26 @@ proc loc_pins {} {
         set tile [get_tiles -of_objects $site]
         set pin [dict get $io_pin_sites $site]
 
-        set_property -dict "PACKAGE_PIN $pin IOSTANDARD LVCMOS33" $port
+        set_property -dict "PACKAGE_PIN $pin IOSTANDARD LVCMOS25" $port
```

Comment:
Toggle between 2.5 and 3.3?

Reply:
Well, the thing is that for LVCMOS33 we will solve the pulltype and slew bits, but not the drive - no clear zero value group problem.

Reply:
A zero value group is not required if it doesn't exist.